nbextensions/advancedViewCell/main.js

```diff
@@ -149,16 +143,11 @@ define([
             var inputPrompt = this.element[0].querySelector('[data-element="prompt"]');
 
             if (inputPrompt) {
-                inputPrompt.innerHTML = div({
-                    style: { textAlign: 'center' }
-                }, [
-                    AppUtils.makeAppIcon(utils.getCellMeta(cell, 'kbase.viewCell.app.spec'))
-                ]);
+                inputPrompt.innerHTML = this.getIcon();
```

Comment:
standardise the way that different cell types render their icons -- now they all use `this.getIcon()` and `getIcon` uses `Icon.makeToolbar(App|Generic|Type)Icon`.